gen_ddp_n8.py

```diff
@@ -180,7 +193,12 @@
         print('Writing {}.'.format(opath_field))
 
         # TODO:  Here we're assuming each GAMA field has 1/3. of the area.
-        to_write_field.meta['AREA'] =  to_write.meta['AREA'] / 3.    
+        # TODO:  Work out DESI area factor (or refactor)
+        if survey == 'gama':
+            to_write_field.meta['AREA'] =  to_write.meta['AREA'] / 3.
```

Comment:
Fine as a TODO for now. 

Suggested approach is we add field_relareas to gama_limits and ros_tools., with {'R1': 1, 'R2': 1} etc.  I.e.  the relative areas to each.